apps/src/lib/script-editor/ScriptEditor.jsx

```diff
@@ -155,6 +162,39 @@ const ScriptEditor = React.createClass({
             style={styles.input}
           />
         </label>
+        <h3>Project widget options</h3>
+        <label>
+          Project widget visible
+          <input
+            name="project_widget_visible"
+            type="checkbox"
+            defaultChecked={this.props.projectWidgetVisible}
+            style={styles.checkbox}
+          />
+          <p>
+            If checked this script will have the projects widget (recent projects and new
+            project buttons) visible in stage extras.
+          </p>
+        </label>
+        <label>
+          Project widget new project types
+          <p>
+            Select up to 4 project type options to appear in the 'Start a new project' section. Select
+            <a className="select_none"> none </a>
```

Comment:
Instead of applying a click handler in `componentDidMount`, why not follow the [React event pattern](https://facebook.github.io/react/docs/handling-events.html) and add an onClick handler to this <a> element that performs the action?

Note you can also use a [ref](https://facebook.github.io/react/docs/refs-and-the-dom.html) for the select control you want to modify.

Something like:
```jsx
<select
  ref={select => this.projectWidgetSelect = select;}
  // ...
>

<a onClick={handleClearProjectWidgetSelectClick}> none </a>
```

And
```
handleClearProjectWidgetSelectClick() {
  $(this.projectWidgetSelect).children('option')[flag ? 'attr' : 'removeAttr']('selected', true);
}
```